src/index.js

```diff
@@ -1,6 +1,7 @@
 import {Options, GLOBAL_OPTIONS, DIRECTION} from './options';
 import {Dragula} from './dragula';
 import {moveBefore} from './move-before';
+// import {PLATFORM} from 'aurelia-framework'; // PLATFORM is already available
```

Comment:
Why is platform already available?  It's _probably_ available (due to most people having the framework installed as part of their project), but I think I would like it to be explicit.  However, as this is a library, I'm trying to keep my dependencies to the minimum required, which is why I don't depend on `aurelia-framework` but rather the specific modules.